services/app-web/src/pages/CMSDashboard/CMSDashboard.tsx

```diff
@@ -214,18 +217,96 @@ const SubmissionsDashboard = (): React.ReactElement => {
 }
 const RateReviewsDashboard = (): React.ReactElement => {
     const { loggedInUser } = useAuth()
-    if (!loggedInUser) {
+    const { data, loading, error } = useIndexRatesQuery({
+        fetchPolicy: 'network-only',
+    })
+
+    const reviewRows: RateInDashboardType[] = []
+    data?.indexRates.edges
+        .map((edge) => edge.node)
+        .forEach((rate) => {
+            const currentRevision = rate.revisions[0]
+            // Set rate display data - could be based on either current or previous revision depending on status
+            let displayRateFormData = currentRevision.formData
+            let lastUpdated = mostRecentDate([
+                currentRevision.submitInfo?.updatedAt,
+                currentRevision.unlockInfo?.updatedAt,
+            ])
+
+            if (rate.status === 'UNLOCKED') {
+                // Errors - data handling
+                const previousRevision = rate.revisions[1]
+                const previousFormData = previousRevision.formData
+
+                // reset package display data since unlock submissions rely on previous revision data
+                displayRateFormData = previousFormData
+                lastUpdated = mostRecentDate([
+                    currentRevision?.submitInfo?.updatedAt,
+                    currentRevision?.unlockInfo?.updatedAt,
+                    previousRevision?.submitInfo?.updatedAt,
+                    previousRevision?.unlockInfo?.updatedAt,
+                ])
+            }
+
+            // Type guards - graphql has loose types with form data, let's narrow in now
+            if (
+                !displayRateFormData ||
+                !displayRateFormData.rateProgramIDs ||
+                !displayRateFormData.rateType ||
+                !displayRateFormData.rateDateEnd ||
```

Comment:
once we have a rate dashboard for states it should be possible to display rates that are incomplete

Reply:
@macrael Are you saying you want our dashboard to display `???` or something like that if the field is missing? We don't have that built in right now - we assume all fields listed here exist on dashboard these are required types for a submitted package. Since we dont have a locked type I believe we need to manually check for missing fields here.

Reply:
Maybe this won't play out that way but it depends on how we end up creating rates. But I can imagine it being possible to create a rate and not finish filling it out and expect it to show up on the index page

Reply:
This already threw an error so anyway shouldn't do the full page takeover. Will stick with logging and check this out more next week.  Have no idea what data was missing since I thought all those fields would be there. Moving to fallback text